ts/Extensions/DashboardPlugins/HighchartsSyncHandlers.ts

```diff
@@ -68,6 +74,29 @@ function getAxisMinMaxMap(chart: Chart): Array<{
         );
 }
 
+/**
+ * Utility function that returns the first row index if the table has been modified by a range modifier
+ * @param {DataTable} table the table to get the offset from
+ */
+function getModifiedTableOffset(table: DataTable): number | undefined {
+    const modifier = table.getModifier();
+    if (modifier && modifier.options.modifier === 'Range') {
+        const { ranges } = (modifier as RangeModifier).options;
+
+        if (ranges) {
+            const minRange = ranges.reduce((minRange, currentRange): RangeModifier.RangeOptions => {
+                if (currentRange.minValue > minRange.minValue) {
+                    minRange = currentRange;
+                }
+                return minRange;
+
+            }, ranges[0]);
+
+            return table.getRowIndexBy(minRange.column, minRange.minValue);
+        }
+    }
+}
```

Comment:
@bre1470 thoughts on where this should go? Does it belong with the modifier class or dataTable? 🤔

Reply:
This should go into RangeModifier, I would say. Maybe during modification it can set a public property in RangeModifier. I will look into easier modifier access, but first ES6 needs a fix.